data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/berserk/berserk_add.cfg

```diff
@@ -0,0 +1,34 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [berserk]value=
+##
+# Actions:
+# Give the leaders a berserk ability with both the value and add attributes
+# the add attribute is ignored
```

Comment:
I think this line should be the first line of "Expect end state", rather than be in the actions. Applies to some other files too.

Reply:
Updated.